src/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -535,7 +535,7 @@ private static void CheckBaseAddress(Uri baseAddress, string parameterName)
 
         private void SetTimeout(CancellationTokenSource cancellationTokenSource)
         {
-            Contract.Requires(cancellationTokenSource != null);
+            Debug.Assert(cancellationTokenSource != null);
```

Comment:
In my first pass I removed the using and project.json entries for them all; but it got a bit upset... As there were other uses.

Will make another pass.